firedrake/interpolation.py

```diff
@@ -33,12 +33,65 @@
 __all__ = (
     "interpolate",
     "Interpolator",
+    "Interpolate",
     "DofNotDefinedError",
     "CrossMeshInterpolator",
     "SameMeshInterpolator",
 )
 
 
+class Interpolate(ufl.Interpolate):
+
+    def __init__(self, expr, v, interp_data=None):
+        r""" Symbolic representation of the interpolation operator.
+
+        :arg expr: a UFL expression to interpolate.
+        :arg v: the :class:`.FunctionSpace` to interpolate into or the :class:`.Coargument`
+                defined on the dual of the :class:`.FunctionSpace` to interpolate into.
+        :arg interp_data: optional dictionary containing interpolation-specific data
```

Comment:
This is not sufficient documentation given this will become the primary API for interpolation. You need to lift the keyword argument list from `Interpolator` and put it here